scenes/game/local_game.gd

```diff
@@ -4132,13 +4188,17 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			events += [create_event(Enums.EventType.EventType_Strike_DodgeFromOppositeBuddy, performing_player.my_id, 0, "", effect['buddy_name'])]
 			_append_log_full(Enums.LogType.LogType_Effect, performing_player, "will dodge attacks from opponents behind %s!" % effect['buddy_name'])
 		"draw":
+			var from_bottom = 'from_bottom' in effect and effect['from_bottom']
 			var amount = effect['amount']
 			if str(amount) == "strike_x":
 				amount = performing_player.strike_stat_boosts.strike_x
 			if amount > 0:
 				if 'opponent' in effect and effect['opponent']:
 					events += opposing_player.draw(amount)
 					_append_log_full(Enums.LogType.LogType_CardInfo, opposing_player, "draws %s card(s)." % amount)
+				if from_bottom:
```

Comment:
What is going on here?
from_bottom should just be passed in for opponent or self, why is there this extra if?